### PR TITLE
VSCode Marketplace Search Command

### LINKED ISSUE #1067: VS Code Marketplace Search Command
https://marketplace.visualstudio.com/vscode

### PR DESCRIPTION
**Pull request by almeidx**
Closes #1067

### CODE REVIEW

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -1629,6 +1629,17 @@
     "lastUpdated": "Last Updated at",
     "notFound": "Code not found"
   },
+  "vscodeextension": {
+    "commandDescription": "Shows information about extensions from the Visual Studio Code Marketplace",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1633]:
```suggestion
    "commandDescription": "Shows information about VS Code extensions from the Visual Studio Marketplace",
```

File: src/locales/en-US/commands.json [hunk 2]
```diff
@@ -1629,6 +1629,17 @@
...
+    "commandUsage": "<name>",
+    "marketplace": "Visual Studio Code Marketplace",
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json:1635]:
```suggestion
    "marketplace": "Visual Studio Marketplace",
```

**Review by pedrofracassi — CHANGES_REQUESTED** [review]

**Review by fernando457829 — APPROVED** [review]

**Comment by fernando457829** [comment]
LGTM

**Review by pedrofracassi — DISMISSED** [review]
LGTM

**Review by fernando457829 — DISMISSED** [review]
LGTM ~again~

**Review by pedrofracassi — APPROVED** [review]
LGTM

**Review by Tsugami — APPROVED** [review]

**Pull request merged**